src/resources/js/views/multiday-events.js

```diff
@@ -0,0 +1,179 @@
+			.find( obj.selectors.multidayEvent + '[data-id=' + eventId + ']' )
+			.not( obj.selectors.hiddenMultidayEvent );
+	};
+
+	/**
+	 * Add class to visible multiday event on hidden multiday event hover
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $visibleMultidayEventInner jQuery object of visible multiday event
+	 *
+	 * @return {function} event handler for on hover in
+	 */
+	obj.onHoverIn = function( $visibleMultidayEventInner ) {
+		return function() {
+			$visibleMultidayEventInner.addClass( obj.selectors.multidayEventInnerHover.className() );
+		};
+	};
+
+	/**
+	 * Remove class to visible multiday event on hidden multiday event hover
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $visibleMultidayEventInner jQuery object of visible multiday event
+	 *
+	 * @return {function} event handler for on hover out
+	 */
+	obj.onHoverOut = function( $visibleMultidayEventInner ) {
+		return function() {
+			$visibleMultidayEventInner.removeClass( obj.selectors.multidayEventInnerHover.className() );
+		};
+	};
+
+	/**
+	 * Add class to visible multiday event on hidden multiday event focus
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $visibleMultidayEventInner jQuery object of visible multiday event
+	 *
+	 * @return {function} event handler for on focus
+	 */
+	obj.onFocus = function( $visibleMultidayEventInner ) {
+		return function() {
+			$visibleMultidayEventInner.addClass( obj.selectors.multidayEventInnerFocus.className() );
+		};
+	};
+
+	/**
+	 * Remove class to visible multiday event on hidden multiday event blur
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $visibleMultidayEventInner jQuery object of visible multiday event
+	 *
+	 * @return {function} event handler for on blur
+	 */
+	obj.onBlur = function( $visibleMultidayEventInner ) {
+		return function() {
+			$visibleMultidayEventInner.removeClass( obj.selectors.multidayEventInnerFocus.className() );
+		};
+	};
+
+	/**
+	 * Binds events for hover and focus of hidden multiday events
+	 *
+	 * @since TBD
+	 *
+	 * @param {number} index index of hidden multiday events from jQuery selector
+	 * @param {HTMLElement} hiddenMultidayEvent HTML element of hidden multiday event
+	 *
+	 * @return {void}
+	 */
+	obj.bindEvents = function() {
+		var $hiddenMultidayEvents = $( obj.selectors.multidayEvent );
+
+		$hiddenMultidayEvents.each( function( index, hiddenMultidayEvent ) {
+			var $hiddenMultidayEvent = $( hiddenMultidayEvent );
+			var $visibleMultidayEvents = obj.findVisibleMultidayEvents( $hiddenMultidayEvent );
+
+			$visibleMultidayEvents.each( function( index, visibleMultidayEvent ) {
+				var $visibleMultidayEvent = $( visibleMultidayEvent );
+				var $visibleMultidayEventInner = $visibleMultidayEvent.find( obj.selectors.multidayEventInner );
+				var $hiddenMultidayEventInner = $hiddenMultidayEvent.find( obj.selectors.multidayEventInner );
+
+				$hiddenMultidayEventInner.hover( obj.onHoverIn( $visibleMultidayEventInner ), obj.onHoverOut( $visibleMultidayEventInner ) );
```

Comment:
Since these are all on one element I would love to have all of those on a single `on()` call. What do you think?

Reply:
not sure how we would do that? the events call different functions.

Reply:
i had to use 2 `on()` calls since they called different functions.